Assets/Scripts/Buddy/BuddyStats.cs

```diff
@@ -66,6 +66,18 @@ enum HappinessState
 	[SerializeField] float _emoteRoutineWait = 0f;
 	Coroutine _currentEmoteRoutine = null;
 
+	[Header( "Debug Settings" )]
+	[SerializeField] bool _disableStatDecrease = false;
+
+	[SerializeField] ParticleSystem _particles;
+	Renderer _particlesRenderer;
+	[SerializeField] ParticleSystem _deadParticleSystem;
```

Comment:
See if it's possible to change the way you're doing this so we don't need a second particle system. It doesn't seem like we're gaining much from having a second one, and it only makes things more complicated.